### PR TITLE
New package: geda-1.8.2

### PR DESCRIPTION
**Pull request by pullmoll**
I chose the all lowercase name, while the project sometimes uses gEDA.

### CODE REVIEW

File: srcpkgs/geda/template [hunk 1]
```diff
@@ -0,0 +1,46 @@
+# Template file for 'geda'
+pkgname=geda
+version=1.8.2
+revision=1
+wrksrc=${pkgname}-gaf-${version}
+build_style=gnu-configure
+hostmakedepends="pkg-config guile desktop-file-utils"
+makedepends="gc-devel gtk+-devel glib-devel guile-devel"
```
Comment (xtraeme) [inline srcpkgs/geda/template]: `gtk+-devel` depends on `libglib-devel` already, remove `glib-devel`.

File: srcpkgs/geda/template [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+depends="guile geda-data>=0"
```
Comment (xtraeme) [inline srcpkgs/geda/template]: `>=0` is redundant here, it's added automagically.

File: srcpkgs/geda/template (hunk 1 above)
Comment (pullmoll) [inline srcpkgs/geda/template]: `xmindep gtk+-devel glib-devel` returns no reduction, but I'll change it and see.

Comment (xtraeme) [inline srcpkgs/geda/template]: - libglib-devel is the pkg that provides the headers and static libs, so symlinks.
- glib-devel contains some utilities to generate glib stuff and depends on libglib-devel.

**Comment by pullmoll** [comment]
For now *-musl remains broken, because I have no fix for guile.

**Pull request merged**